README.md

```diff
@@ -147,6 +148,46 @@ The only directive currently supported is `FormatBlockComment` and it's used as
 ;@AHK++FormatBlockCommentOff
 ```
 
+#### Formatter Known Issues
+
+There are commands, that can produce indent on single next line. For example:
+
+```autohotkey
+if (true)
+    MsgBox
+
+Loop % n
+    SoundBeep
+```
+
+Such code will be well formatted.
+
+But do not nest such commands. The next code will be formatted incorrectly:
```

Comment:
Let's file an issue for this and add it in the README as well

Reply:
In fact, we can just link the search results for all bugs labelled "formatter" so folks can easily learn about current active issues.

Reply:
The link to the bugs will probably be better than describing this bug in detail. Let's consider replacing this PR with one that just adds a link to all formatter bugs instead to keep the README short and simple.